cmd/podman/images.go

```diff
@@ -21,16 +21,16 @@ import (
 )
 
 type imagesTemplateParams struct {
-	Repository  string
-	Tag         string
-	ID          string
-	Digest      digest.Digest
-	Digests     []digest.Digest
-	Created     string
-	CreatedTime time.Time
-	Size        string
-	ReadOnly    bool
-	History     string
+	Repository   string
+	Tag          string
+	ID           string
+	Digest       digest.Digest
+	Digests      []digest.Digest
+	CreatedAt    time.Time
+	CreatedSince string
+	Size         string
+	ReadOnly     bool
+	History      string
```

Comment:
Just wondering if we should still support "Created" and "CreatedTime" in case people have adjusted and have built dependencies on those?  I'd be fine if they were hidden options going forward, available for use, but not documented.

Reply:
My concern is that keeping `Created` and `CreatedTime` in this struct for hidden options seems duplicate.

Reply:
yep, it's certainly a duplicate and I'd add a comment in the code as to why it's there.  My concern is someone out there has already built a dependency on one of those and we might break them if they go away.

Reply:
I would not retain duplicates in the structures - especially given how similar all these names are. We may want to look into normalization similar to what we're doing in `podman inspect --format` though.

Reply:
@TomSweeneyRedHat @mheon Thanks. I will do that.

Reply:
If Matt's hip, then I'm hip.  Thanks for the discussion!